### ISSUE TITLE
Could not find 'UnityEngine.TextMesh' in IL2CPP domain even though it could be found in the managed domain.

### ISSUE THREAD

**Issue by senngoku** [bbepis/XUnity.AutoTranslator#550]
I am trying to translate a game called Jurnal Malam : Bestfirend using melonloader but it is not working. I have looked at the following thread and tried everything but it did not work, so I need your help on how to proceed.
#287.
#57
#351

I will also leave the last log. Thank you in advance for your help.

[Latest.log](https://github.com/user-attachments/files/16227538/Latest.log)
[config.txt](https://github.com/user-attachments/files/16227578/config.txt)

**Comment by Atamg1994** [comment]
https://github.com/bbepis/XUnity.AutoTranslator/issues/287.
https://github.com/bbepis/XUnity.AutoTranslator/issues/57
https://github.com/bbepis/XUnity.AutoTranslator/issues/351
This solution was made for version 0.5.7 and translation tool 5.2.0

new versions use other libraries and the solution doesn’t necessarily work

**Comment by senngoku** [comment]
Thank you for your reply.

I have just tried with melonloader v0.5.7 and translation tool v5.2.0, but at the end it logged that it is not supported. The logs seem to show a lot of errors. I also started the game but it did not translate.
Is there any way to fix this?
Thank you for your attention. Thank you very much for your help.

[MelonLoader_24-07-14_23-49-03.892.log](https://github.com/user-attachments/files/16227816/MelonLoader_24-07-14_23-49-03.892.log)

**Comment by Atamg1994** [comment]
> Thank you for your reply.
> 
> I have just tried with melonloader v0.5.7 and translation tool v5.2.0, but at the end it logged that it is not supported. The logs seem to show a lot of errors. I also started the game but it did not translate. Is there any way to fix this? Thank you for your attention. Thank you very much for your help.
> 
> [MelonLoader_24-07-14_23-49-03.892.log](https://github.com/user-attachments/files/16227816/MelonLoader_24-07-14_23-49-03.892.log)

yes.. indeed this game does not work with old versions... then you need to rewrite the solution itself for new versions... but I don’t have time for that... in fact, this solution was cut from a huge mod... I cut it for myself and my needs, you can also try to dig into the library yourself with the dnspy ​​tool

**Comment by senngoku** [comment]
Thank you for your reply.

I tried other recent games, but they didn't work so well, so maybe newer versions need to be made for those versions. In the meantime, I have tried it before and will try again with the newer, more recent versions.

Thank you for listening. Thank you very much for your kind attention.
